docs/src/operations/requirements.md

```diff
@@ -18,12 +18,19 @@ transaction for each block the validator agrees with, which can cost up to
 
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
+**NOTE**: These recommendations may be out of date as the network grows and resource requirements increase.
```

Comment:
Maybe say "change" instead of increase? Better future-proofed.

Reply:
Good call, fixed in cf8d6dbe7891746c9b46d8f173d9424687961d72